pkgs/applications/video/f1viewer/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGithub }:
+
+buildGoModule rec {
+  pname = "f1viewer";
+  version = "2.4.0";
+
+  src = fetchFromGitHub {
+    owner = "SoMuchForSubtlety";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "7eXRUG74l9+9nU7EmDvNcHc+2pg5+/amjqtrzT60f94=";
+  };
+
+  vendorSha256 = "4pQ8NT0mh3w7naHEHh2w6Csop0uitlWClZ95VlYaPW0=";
+
+  meta = with lib; {
+    description = "TUI for F1TV.";
```

```suggestion
    description = "TUI for F1TV";
```
https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes

This should also be more descriptive.

---

My apologies for the missing period. I had added it, not sure where it got lost. I gave that description because that is the description the author wrote for the project's repository itself and I thought it best to stick to source.

---

My apologies for the period, I had some confirmation bias when I read CONTRIBUTING.md and just assumed the period was desired.

---

The description itself is what it is because that is what the author gives in the projects repository. I thought it best to stick to source, but I could try to think something up if required.

---

How about 'An application to view Formula 1 races and highlights without a browser, using VLC or other media player.' ?

---

I also will note that the period has been removed from the proposed changes, keeping this conversation un-resolved to reach resolution on the contents of the description itself.